nextflow.config

```diff
@@ -26,7 +28,11 @@ params {
     fastp_additional_params = '-5 -3 -W 4 -M 20 -l 15 -x -n 5 -z 6'
     histat2_additional_params = ''
 
+    busco = 'euarchontoglires_odb9'
+    full = ''
```

Comment:
I'd like something more descriptive here. maybe `UniRef90` ?

Reply:
good point, I change this to `params.uniref90`

Reply:
I also renamed the output folder from `dammit/full` to `dammit/uniref90`. I think there are other parts that could be simplified... but yeah should be fine for now. I will test the download still works properly for the full db (uniref90 is insanely large)